pkgs/data/fonts/newcomputermodern/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "newcomputermodern";
+  version = "4.4";
+
+  src = fetchzip {
+    url = "https://mirrors.ctan.org/fonts/newcomputermodern.zip";
```

Comment:
This is not a stable url. Maybe use https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2022/tlnet-final/archive/newcomputermodern.doc.r66338.tar.xz (or another mirror from https://tug.org/historic/ ) instead?

newcomputermodern is also already package as part of texlive, so just repackaging this in a more convenient format (similar to https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/iwona/default.nix ) is also an option.

Reply:
Excellent ! I didn't know that. I updated the PR.